mobile/src/core/store/app/actions.ts

```diff
@@ -16,7 +17,8 @@ export const loadApp = createActionFn<void, Promise<void>>(
       dispatch(loadAccounts()),
       dispatch(loadPasscode()),
       dispatch(loadPasscodeEnteredTime()),
-      dispatch(loadAppSettings())
+      dispatch(loadAppSettings()),
+      dispatch(loadCMCData())
```

Comment:
It's network-based thing and app can work without it properly, so i don't think we need to wait for it at least here.